### DISCUSSION TITLE
how should i import tempo v2?

### DISCUSSION THREAD

**Discussion by zdyj3170101136** [grafana/tempo#3112] [Q&A]
when import github.com/grafana/tempo i only could get v1.5.0.

how should i import v2?

**Comment by joe-elliott** [comment]
This is due to the way that go does versioning from 2.x forward. I believe you need to import a specific commit for it to work:

```
go get github.com/grafana/tempo@<some commit hash>Z
```

**Comment by boris-smidt-klarrio** [comment]
Could the project actually be made 'compatible' with the go convention to avoid import problems like this?

**Reply by joe-elliott** [reply]
I'm not opposed to it, but it's definitely not a priority for us. Someone attempted this awhile back and I believe we dropped the ball:

https://github.com/grafana/tempo/pull/3926
